.github/workflows/codeql.yml

```diff
@@ -0,0 +1,42 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches:
+      - master
+      - version/*
+  pull_request:
+    branches:
+      - master
+      - version/*
```

Comment:
Is this branch pattern active?

Reply:
I was not entirely sure.  I only see one branch with the pattern but I figured it was safer to include than to miss it